app/Makefile

```diff
@@ -134,9 +233,59 @@ $(BUILDDIR)/%.o: %.cpp | build_directories
 
 build_directories:
```

Comment:
mh ill have to look if there is a better way to solve this

Reply:
in the meantime i think its good to merge this first, so we can go ahead with the fishtest integration